packages/sqlite/test/libsql.test.ts

```diff
@@ -0,0 +1,502 @@
+import { Cart, ContactPreferences } from "../../test-domain/src/index.js"
+import { Decider, MemoryCache, Codec, CachingStrategy, Tags, StreamId } from "@equinox-js/core"
+import { describe, test, expect, afterEach, afterAll, vi, beforeAll } from "vitest"
+import { randomUUID } from "crypto"
+import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node"
+import { InMemorySpanExporter, SimpleSpanProcessor } from "@opentelemetry/sdk-trace-base"
+import fs from "fs"
+import {
+  AccessStrategy,
+  LibSqlCategory,
+  LibSqlConnection,
+  LibSqlContext,
+  initializeDatabase,
+} from "../src/index.js"
+import { createClient } from "@libsql/client"
+import { trace } from "@opentelemetry/api"
+
+const Category = LibSqlCategory
+
+const defaultBatchSize = 500
+
+namespace CartService {
+  type E = Cart.Events.Event
+  type S = Cart.Fold.State
+  const codec = Cart.Events.codec
+  const fold = Cart.Fold.fold
+  const initial = Cart.Fold.initial
+  const cache = new MemoryCache()
+  const noCache = CachingStrategy.NoCache()
+
+  export function createWithoutOptimization(context: LibSqlContext) {
+    const category = Category.create(
+      context,
+      Cart.Category,
+      codec,
+      fold,
+      initial,
+      noCache,
+      AccessStrategy.Unoptimized(),
+    )
+    return Cart.Service.create(category)
+  }
+
+  export function createWithSnapshotStrategy(context: LibSqlContext) {
+    const access = AccessStrategy.AdjacentSnapshots<E, S>(
+      Cart.Fold.snapshotEventType,
+      Cart.Fold.snapshot,
+    )
+    const category = Category.create(context, Cart.Category, codec, fold, initial, noCache, access)
```

Comment:
CategoryName these days

Reply:
This is just tests, in proper code it'd be `Cart.Stream.Category` which I think is specific enough given the namespace. I'm not quite sold on the `Name` postfix yet

Reply:
I removed Stream from the F# in full and life is quite good.
One driver is that Category clashes with lots of things in messy ways with shadowing (and Stream can potentially do that too)
Obv the Stream module and how it works in TS is good so Stream.Category vs CategoryName can be fine. Probably as long as Category is not used without the prefix, I guess the ambiguity is not to the same degree.